### ISSUE TITLE
SmartCity-Accessibility #338 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#199]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 168 lines...]
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078436 lng is : 34.78706
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.044 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.navigation.NavigationTest
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.402 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  ReviewManagerTest.getReviewByUserAndLocationTest:80 » IndexOutOfBounds Index: ...

Tests run: 52, Failures: 0, Errors: 1, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:58 min
[INFO] Finished at: 2017-01-21T14:57:21+00:00
[INFO] Final Memory: 30M/152M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] c42e4439e41374c8c83bf64e36bf92e22b7dd3f9 - fixed bug with pinned reviews

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/338/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 168 lines...]
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
lat is : 31.901406 lng is : 35.200473
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.058 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.navigation.NavigationTest
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078436 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.567 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  ReviewManagerTest.getReviewByUserAndLocationTest:80 » IndexOutOfBounds Index: ...

Tests run: 52, Failures: 0, Errors: 1, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:28 min
[INFO] Finished at: 2017-01-21T16:40:32+00:00
[INFO] Final Memory: 28M/180M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] c42e4439e41374c8c83bf64e36bf92e22b7dd3f9 - fixed bug with pinned reviews

- [yaelAmitay] b473159d9a6f9db653113388ef99243ad345c43b - fix number of stars in the ratig



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/339/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 168 lines...]
lat is : 31.756353 lng is : 35.221459
lat is : 31.770624 lng is : 35.221814
lat is : 31.784296 lng is : 35.228537
lat is : 31.795226 lng is : 35.068592
lat is : 32.067312 lng is : 34.7781
lat is : 32.061621 lng is : 34.764118
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.072 sec - in smartcity.accessibility.search.NearbyPlacesTest
Running smartcity.accessibility.navigation.NavigationTest
lat is : 31.901406 lng is : 35.200473
lat is : 31.777545 lng is : 35.221896
lat is : 32.060276 lng is : 34.764359
lat is : 32.078436 lng is : 34.78706
lat is : 32.061957 lng is : 34.783662
lat is : 31.784352 lng is : 35.117952
lat is : 31.778306 lng is : 35.216163
lat is : 32.124488 lng is : 34.825184
lat is : 32.108255 lng is : 34.838057
lat is : 32.069621 lng is : 34.793664
lat is : 32.077567 lng is : 34.789139
lat is : 32.114059 lng is : 34.824801
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.318 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  ReviewManagerTest.getReviewByUserAndLocationTest:80 » IndexOutOfBounds Index: ...

Tests run: 52, Failures: 0, Errors: 1, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:00 min
[INFO] Finished at: 2017-01-21T16:59:58+00:00
[INFO] Final Memory: 28M/201M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] c42e4439e41374c8c83bf64e36bf92e22b7dd3f9 - fixed bug with pinned reviews

- [yaelAmitay] b473159d9a6f9db653113388ef99243ad345c43b - fix number of stars in the ratig

- [yaelAmitay] 64eae544e1c5afdf8e5f16c8be0098dcfcd82c46 - add specific error message to navigation



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/340/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
